backend/API_DOC_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Template
+
+## API documentation template
+
+Copy and paste below template when creating a new endpoint documentation.
+
+### Endpoint Name
+
+Endpoint description here.
+
+**URL** : `http://api.good-grocer.click/ENDPOINT_URL_HERE`
+
+**Method** : `GET|POST|PUT|DELETE`
+
+**Auth required** : NO
+
```

Comment:
Might be helpful to add a section in the template for query parameters or URI parameters